js_modules/dagit/packages/core/src/partitions/SpanRepresentation.tsx

```diff
@@ -83,8 +81,10 @@ export function spanTextToSelections(
         start = -1;
       };
 
-      for (let idx = 0; idx < all.length; idx++) {
-        const match = all[idx].startsWith(prefix) && all[idx].endsWith(suffix);
+      // todo bengotow: Was this change correct??
+      for (let idx = 0; idx < allPartitionKeys.length; idx++) {
+        const match =
+          allPartitionKeys[idx].startsWith(prefix) && allPartitionKeys[idx].endsWith(suffix);
```

Comment:
@bengotow Do you know if this is/was correct? `all` was being pulled in as an import, but it didn't really seem to make sense here.